db_task2/dbOperation.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+function getElectiveByID($electiveID){
+    try{
+        $conn=connection();
+        $stmt = $conn->prepare("SELECT * FROM electives WHERE id=:idPlaceholder limit 1;");
+        $stmt->bindParam(':idPlaceholder', $electiveID);
+        $stmt->execute();
+        return $stmt->fetch(PDO::FETCH_ASSOC);
+    }
+    catch(PDOException $error) {
+        die($error->getMessage());
+    }
+}
+
+
+function updateElectiveByID($electiveID){
+    try{
+        $conn=connection();
+        $courseTitle = $_POST['title'];
+        $name = $_POST['name'];
+        $description = $_POST['description'];
+        $stmt=$conn->prepare("UPDATE electives SET title=:titlePlaceholder, description=:descriptionPlaceholder, lecturer=:lecturerPlaceholder WHERE id=:idPlaceholder;");
+        $stmt->bindParam(':idPlaceholder', $electiveID);
+        $stmt->bindParam(':titlePlaceholder', $courseTitle);
+        $stmt->bindParam(':descriptionPlaceholder', $description);
+        $stmt->bindParam(':lecturerPlaceholder', $name);
+        $stmt->execute();
```

Comment:
You may want to proactively close the sql connections when you finish with them.
https://www.w3schools.com/php/php_mysql_connect.asp

https://stackoverflow.com/a/15444763/3837638

>it's not pointless to close a connection. Every connection occupies resources in the database server. Most servers are configured with a maximum number of concurrent connections. If you close connections proactively, you free up that slot for another thread, and you get overall better throughput.

This applies to all places where you create a new connection and use it.